porcupine/_state.py

```diff
@@ -4,13 +4,16 @@
 import tkinter
 import types
 from typing import Any, Dict, Optional, Type
+from tkinter import ttk
+
 
 from porcupine import tabs
 
 log = logging.getLogger(__name__)
 
 # global state makes some things a lot easier
 _root: Optional[tkinter.Tk] = None
+_pw: Optional[tkinter.PanedWindow] = None
```

Comment:
```suggestion
_pw: Optional[ttk.Panedwindow] = None
```

Might fix mypy problem